frontend/shared/components/src/setupSteps/SetupStepRow.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <STListItem class="left-center right-stack" :selectable="isSelectable">
+        <template #left>
+            <IconContainer :class="color" :icon="icon">
+                <template #aside>
+                    <ProgressIcon v-if="$secondaryIcon || $progress" :icon="$secondaryIcon" :progress="$progress" />
+                </template>
+            </IconContainer>
+        </template>
+        <h2 class="style-title-list">
+            {{ title }}
+        </h2>
+        <p v-if="listType === 'todo'" class="style-description">
+            {{ description }}
+        </p>
+        <p v-else class="style-description-small">
+            {{ reviewDescription }}
+        </p>
+        <template #right>
+            <span v-if="!step.isDone" class="style-description-small">{{ step.finishedSteps }} / {{ step.totalSteps }}</span>
+            <span v-if="isSelectable" class="icon arrow-right-small gray" />
+        </template>
+    </STListItem>
+</template>
+
+<script setup lang="ts">
+import { IconContainer, ProgressIcon, STListItem, useSetupStepReviewDescription, useSetupStepTranslations } from '@stamhoofd/components';
+import { useTranslate } from '@stamhoofd/frontend-i18n';
+import { SetupStep, SetupStepType } from '@stamhoofd/structures';
+import { computed } from 'vue';
+
+const props = defineProps<{setupStepType: SetupStepType, step: SetupStep, listType: 'review' | 'todo'}>();
```

Comment:
```suggestion
const props = defineProps<{
    setupStepType: SetupStepType, 
    step: SetupStep, 
    listType: 'review' | 'todo'
}>();
```

Best props altijd consequent op dezelfde manier formatteren voor de leesbaarheid